h2o-py/h2o/backend/server.py

```diff
@@ -477,7 +477,7 @@ def _get_server_info_from_logs(self):
 
         :returns: (scheme, ip, port) tuple if the server has already started, None otherwise.
         """
-        searchstr = "INFO: Open H2O Flow in your web browser:"
+        searchstr = ": Open H2O Flow in your web browser:"
```

Comment:
@michalkurka this means that the python package is not backwards compatible with previous H2O releases

Reply:
ok, I think this should not go into the release branch then - we also need to make sure if Steam is able to handle it (it used to parse the same String... could be different now)